frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -242,12 +242,19 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
 
     deleteTranslation = (translationId: number) => {
         const { parameters, pluralForm, dispatch } = this.props;
-        dispatch(history.actions.deleteTranslation(
-            parameters.entity,
-            parameters.locale,
-            pluralForm,
-            translationId,
-        ));
+        dispatch(
+            unsavedchanges.actions.check(
+                this.props.unsavedchanges,
+                () => {
+                    dispatch(history.actions.deleteTranslation(
+                        parameters.entity,
+                        parameters.locale,
+                        pluralForm,
+                        translationId,
+                    ));
+                }
+            )
+        );
```

Comment:
This one doesn't change the content of the editor, as far as I can tell, so it's not needed to run the unsaved actions check here.

Reply:
Correct :)

Reply:
you are right, fixed 👍